.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Quest-Tutorials"]
+	path = docs/_tutorials
+	url = https://github.com/BetonQuest/Quest-Tutorials
```

Comment:
use ssh instead

Reply:
I don't use ssh to log into git so ssh urls won't work for me.

Reply:
but ssh urls are read only, and you need to use ssh instead to be able to do edits

Reply:
idk what we are talking about. Some users prefer authentication over ssh, some over https. [Github recommends https](https://docs.github.com/en/get-started/quickstart/set-up-git#authenticating-with-github-from-git).

We could try to use a relative url as suggested in https://stackoverflow.com/questions/6031494/git-submodules-and-ssh-access to solve this protocol issue.

Reply:
If this would work this would be perfect